jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -589,18 +593,23 @@ else if (failure instanceof BadMessageException | failure instanceof IOException
 
     private void minimalErrorResponse(Throwable failure)
     {
-        try
-        {
-            Integer code=(Integer)_request.getAttribute(RequestDispatcher.ERROR_STATUS_CODE);
-            _response.reset(true);
-            _response.setStatus(code == null ? 500 : code);
-            _response.flushBuffer();
-        }
-        catch (Throwable x)
-        {
-            if (x != failure)
-                failure.addSuppressed(x);
+        if (_response.isCommitted())
```

Comment:
I don't think this is needed. If the response is already committed, the write below would fail and we'll `abort()`.